as/cNFT/assembly/models/persistent_nft_contract_metadata.ts

```diff
@@ -17,6 +17,7 @@ export class NFTContractMetadata {
     render_script: string
     style_css: string
     parameters: string
+    nft_per_owner: number
```

Comment:
could you merge master and move this, and all props that are not in NFTContractMetadata standard inside extra to keep them separated. Thanks!